privacyidea/models.py

```diff
@@ -1525,7 +1525,7 @@ def __init__(self, machinetoken_id, key, value):
                                                             machinetoken_id))
         self.machinetoken_id = machinetoken_id
         self.mt_key = key
-        self.mt_value = unicode(value)
+        self.mt_value = value
```

Comment:
Hm, without the ``unicode()`` we trust that ``mt_value`` will always be a unicode string already. Maybe we want to use ``convert_column_to_unicode`` as a safeguard here?

Reply:
Yes. I forgot about ``convert_column_to_unicode``.

Reply:
fixed in 40beef11